elementary/monitor/alerts/group_of_alerts.py

```diff
@@ -214,6 +214,7 @@ def _get_model_error_block_header(self) -> List:
             return []
         result = []
         for model_error_alert in model_error_alert_list:
+            assert isinstance(model_error_alert, ModelAlert)
```

Comment:
How could a non `ModelAlert` be in this list in the first place?
Maybe we should make a check and raise an error?

Reply:
The problem is that `self._components_to_alerts` is a dict whose value changes according to the provided key. There's no way that I can tell to make mypy understand it.
Changed to an exception though.

Reply:
(may worth refactoring, but I want to avoid it for now)